packages/client/src/components/Group/components/Chat/Chat.tsx

```diff
@@ -164,6 +166,19 @@ export default function Chat(props: Props) {
     });
   };
 
+  const onGiphyPickerExited = () => {
+    // HACK: doesn't focus right without it ¯\_(ツ)_/¯
```

Comment:
looks like the setTimeout is commented out, do we still need it?

Reply:
nope I can remove it should I remove it

Reply:
yeah, if we dont need it might as well pop it out